other/materials_designer/create_interface_with_no_strain_matching.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
<h1>Create an interface between two materials</h1><p>This notebook demonstrates how to create interface between two bulk materials of which one is designated as <code>film</code> (top)  and the other one as <code>substrate</code> (bottom). </p><p> </p><p>Summary of the steps below:</p><ol><li>Make sure to select Input Materials</li><li>Set notebook parameters</li><li>Create slab configurations for film and substrate from bulk materials and slab parameters</li><li>View possible slab terminations and select the termination pair for the interface</li><li>Create interface by wrapping film slab cell into the substrate slab cell</li><li>Visualize the interface</li><li>Pass the interface to the outside runtime</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>